src/awfy.js

```diff
@@ -256,7 +256,8 @@ const TALOS_TESTS = {
 
 const TALOS_BENCHMARKS = {};
 Object.entries(TALOS_TESTS).forEach(([testKey, test]) => {
-  TALOS_BENCHMARKS[testKey] = { compare: {}, label: test.label || testKey };
+  const docUrl = `https://firefox-source-docs.mozilla.org/testing/perfdocs/talos.html#${testKey.replaceAll('_', '-')}`;
```

Comment:
There is a small issue here and it fails when testing with jest, but works when trying it live. It's the same problem described here: https://stackoverflow.com/questions/65295584/jest-typeerror-replaceall-is-not-a-function

`testKey.replaceAll('_', '-')` just has to be replaced with `testKey.replace(/_/g, '-')` to not upset node.